### DISCUSSION TITLE
Contact devblogs.microsoft.com

### DISCUSSION THREAD

**Discussion by Rand-Random** [dotnet/runtime#82289] [General] [answered]
How to contact someone responsible on devblogs.microsoft.com ?
I would like to appeal my ban on this platform, but any way to contact on this platform requires me to log-in which I can't because of said ban.

@jongalloway should be the person responsible for the ban, as I tried to point out on this blog post https://devblogs.microsoft.com/dotnet/announcing-dotnet-maui-beautiful-ui-challenge/ that a multi-billion $ company only can afford sticker packs, and since my critizsm was deleted in stead of confronting it, I called out the deleting as pathetic

so half a year later I would like to appeal this ban

**Comment by jongalloway** [comment, marked as answer]
Sorry, deleting a comment shouldn't have locked your account. It's possible that I deleted the comment as spam, and that flagged it. I tried to look up your account from the post comments, but don't see it. Please e-mail me (first name . last name at microsoft . com) so I can get your account info and pass it to the blog admin team to get this fixed.

**Reply by jongalloway** [reply]
Alternatively, if you'd prefer not to work with me, let me know and I can locate a support e-mail address for the blog.

**Reply by Rand-Random** [reply]
@jongalloway
thanks, for taking your time, I don't have an issue working with you
since my account info aren't a mystery, I feel comfortable sharing them here as they are basically my e-mail (try to keep my real name private)

there for
first name is: Rand 
last name is: Random
mail: rand.random@outloook.com

**Reply by jongalloway** [reply]
Still checking on this... sorry it's taking a while.

**Reply by Rand-Random** [reply]
Take your time, what are a couple days/weeks when I only cared to ask after half a year. :)

**Reply by jongalloway** [reply]
Should be unbanned now, let me know if it doesn't work.

**Reply by Rand-Random** [reply]
Yes, I can login in again, thanks
